.github/workflows/integration.yml

```diff
@@ -0,0 +1,16 @@
+# This workflow runs a schedule and uses the latest version of tt-mlir (origin/main branch)
+# to check if the latest version of tt-mlir is compatible with the current codebase.
+
+name: Integration
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
```

Comment:
Is it possible to automatically create PR when this job passes successfully?

Reply:
I'll look into it, there are a few options. What exactly is the goal here?
Create PR to merge submodule update for third_party/tt-mlir to whatever version we had as input in this workflow?
Assign this to someone specific for review and approval? 
Or merge the uplift automatically in the true CI fashion?

Reply:
Main goal would be to have automatic uplift once push (in future full nightly) CI passes. 

For now, it's good enough to create PR and assign me and @sdjordjevicTT to approve and merge once we confirm everything is in place. After few iterations, we can see to push this automatically e2e.